test/unit/test_environment.py

```diff
@@ -47,6 +48,7 @@ def test_env():
     assert env.management_http_port == "1738"
     assert env.safe_port_range == "1111-2222"
     assert "-XX:-UseContainerSupport" in env.vmargs
+    assert env.max_request_size == 10485760
```

Comment:
it is better to use 10 *1024*1024 instead of 10485760

Reply:
agree, updated.